src/content/index.ts

```diff
@@ -38,7 +38,7 @@ function getSelectorsByPlatform(
         chatContainerSelector: ".chat-scrollable-area__message-container",
         chatMessageAuthorSelector:
           ".chat-line__message .chat-author__display-name",
-        chatMessageAuthorBadgeSelector: ".chat-line__message .chat-badge",
+        chatMessageAuthorBadgeSelector: ".chat-badge",
```

Comment:
This is fine, but keep in mind, it's better if we are specific, otherwise, we will traverse the DOM more then we should.

Reply:
When I tested it, it seemed to me that the "chat-line__message" selector was miessing up. but I was wrong and you're right. I brought "chat-line__message" back.